ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -2686,16 +2698,19 @@ msgstr "Données disponibles"
 #: ckanext/ontario_theme/templates/package/snippets/ontario_theme_access_level.html:11
 #, python-format
 msgid ""
-" The data described here is available for you to use. <a "
+"The data described here is available for you to use. <a "
 "href=\"https://www.ontario.ca/page/ontarios-open-data-directive\">[Learn "
 "more]</a><br /> <a href=\"%(license_url)s\">[See the licence for how you're "
-"allowed to use this data.]</a> "
+"allowed to use this data.]</a>"
 msgstr ""
 " Les données décrites ici sont disponibles pour votre utilisation. <a "
-"href=\"https://www.ontario.ca/page/ontarios-open-data-directive\">[Apprendre "
```

Comment:
Since this link is used pretty often, you can create a variable using `{{ _('https://www.ontario.ca/page/ontarios-open-data-directive') }}` and translate the link once in the .po file.

Reply:
It would mean ending (`{% endtrans %}`) the translation before the `<a>` and wrapping the text inside the `<a>` with a new translation (`{% trans %}{% endtrans %}`). The current way works as well, it's up to you!